src/main/java/frc/robot/subsystems/Passer.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.NeoMotorConstants;
+import frc.robot.constants.PasserConstants;
+
+public class Passer extends SubsystemBase {
+    private final CANSparkMax passerMotor = new CANSparkMax(
+            PasserConstants.passerMotorPort, MotorType.kBrushless);
+    private final RelativeEncoder passerEncoder = passerMotor.getEncoder();
+
+    private static Passer instance;
+
+    /**
+     * Creates an instance of the Passer subsystem if it does not already exist.
+     *
+     * @return An instance of the Passer subsystem.
+     */
+    public static Passer getInstance() {
+        if (instance == null) {
+            instance = new Passer();
+        }
+        return instance;
+    }
+
+    private Passer() {
+        passerMotor.restoreFactoryDefaults();
+        passerMotor.setIdleMode(IdleMode.kBrake);
```

Comment:
Probably should be coast like shooter

Reply:
isn't shooter brake? or is that a different motor than the roller motor?

Reply:
Maybe I’m wrong but I thought it was in coast. If not this is fine